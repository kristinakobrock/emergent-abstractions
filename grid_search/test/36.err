  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  4%|###7                                                                                         | 2/50 [00:00<00:02, 19.59it/s] 10%|#########3                                                                                   | 5/50 [00:00<00:02, 21.92it/s] 16%|##############8                                                                              | 8/50 [00:00<00:01, 23.25it/s] 22%|####################2                                                                       | 11/50 [00:00<00:01, 25.36it/s] 28%|#########################7                                                                  | 14/50 [00:00<00:01, 23.79it/s] 36%|#################################1                                                          | 18/50 [00:00<00:01, 26.32it/s] 44%|########################################4                                                   | 22/50 [00:00<00:01, 27.46it/s] 50%|##############################################                                              | 25/50 [00:01<00:01, 24.46it/s] 58%|#####################################################3                                      | 29/50 [00:01<00:00, 28.33it/s] 64%|##########################################################8                                 | 32/50 [00:01<00:00, 26.41it/s] 70%|################################################################3                           | 35/50 [00:01<00:00, 26.19it/s] 76%|#####################################################################9                      | 38/50 [00:01<00:00, 23.96it/s] 84%|#############################################################################2              | 42/50 [00:01<00:00, 25.46it/s] 90%|##################################################################################8         | 45/50 [00:01<00:00, 25.51it/s] 96%|########################################################################################3   | 48/50 [00:01<00:00, 25.55it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.06it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 25.59it/s] 54%|##################################################                                           | 7/13 [00:00<00:00, 28.23it/s] 77%|######################################################################7                     | 10/13 [00:00<00:00, 25.57it/s]100%|############################################################################################| 13/13 [00:00<00:00, 26.96it/s]100%|############################################################################################| 13/13 [00:00<00:00, 26.79it/s]
