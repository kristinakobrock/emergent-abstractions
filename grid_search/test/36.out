# ["--attributes=3", "--values=16", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=64", "--learning_rate=0.001", "--hidden_size=128", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6697797775268555, "acc": 0.6173209547996521, "length": 3.502375364303589, "mode": "train", "epoch": 1}
{"loss": 0.6721858978271484, "acc": 0.6070312857627869, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6622837781906128, "acc": 0.6260793805122375, "length": 3.4272189140319824, "mode": "train", "epoch": 2}
{"loss": 0.6660645604133606, "acc": 0.6204864382743835, "length": 1.01171875, "mode": "test", "epoch": 2}
{"loss": 0.6596903204917908, "acc": 0.6271536350250244, "length": 3.145447254180908, "mode": "train", "epoch": 3}
{"loss": 0.6531210541725159, "acc": 0.620837390422821, "length": 2.1015625, "mode": "test", "epoch": 3}
{"loss": 0.6483818888664246, "acc": 0.6316925287246704, "length": 3.0785744190216064, "mode": "train", "epoch": 4}
{"loss": 0.6316803097724915, "acc": 0.6409362554550171, "length": 2.93359375, "mode": "test", "epoch": 4}
{"loss": 0.6176795959472656, "acc": 0.6588515639305115, "length": 3.088484048843384, "mode": "train", "epoch": 5}
{"loss": 0.6204623579978943, "acc": 0.6547821164131165, "length": 3.57421875, "mode": "test", "epoch": 5}
{"loss": 0.5853595733642578, "acc": 0.6892685890197754, "length": 3.5903115272521973, "mode": "train", "epoch": 6}
{"loss": 0.5933785438537598, "acc": 0.7003264427185059, "length": 3.78125, "mode": "test", "epoch": 6}
{"loss": 0.5569353699684143, "acc": 0.7133010625839233, "length": 3.6844260692596436, "mode": "train", "epoch": 7}
{"loss": 0.5784342288970947, "acc": 0.7078827023506165, "length": 3.90234375, "mode": "test", "epoch": 7}
{"loss": 0.5369406342506409, "acc": 0.7283377051353455, "length": 3.7840044498443604, "mode": "train", "epoch": 8}
{"loss": 0.5602204203605652, "acc": 0.7314392924308777, "length": 3.9609375, "mode": "test", "epoch": 8}
{"loss": 0.5209844708442688, "acc": 0.7384670972824097, "length": 3.8491814136505127, "mode": "train", "epoch": 9}
{"loss": 0.5375671982765198, "acc": 0.7455596923828125, "length": 3.95703125, "mode": "test", "epoch": 9}
{"loss": 0.5112010836601257, "acc": 0.7426847219467163, "length": 3.8892714977264404, "mode": "train", "epoch": 10}
{"loss": 0.5244465470314026, "acc": 0.749035656452179, "length": 3.96875, "mode": "test", "epoch": 10}
{"loss": 0.49956777691841125, "acc": 0.751753032207489, "length": 3.8921520709991455, "mode": "train", "epoch": 11}
{"loss": 0.5231629014015198, "acc": 0.74853515625, "length": 3.95703125, "mode": "test", "epoch": 11}
{"loss": 0.48935624957084656, "acc": 0.7622858285903931, "length": 3.899719476699829, "mode": "train", "epoch": 12}
{"loss": 0.517867386341095, "acc": 0.7466980218887329, "length": 3.9765625, "mode": "test", "epoch": 12}
{"loss": 0.4755086898803711, "acc": 0.7717666625976562, "length": 3.9347214698791504, "mode": "train", "epoch": 13}
{"loss": 0.5020389556884766, "acc": 0.7649964094161987, "length": 3.98828125, "mode": "test", "epoch": 13}
{"loss": 0.4624306559562683, "acc": 0.7779127359390259, "length": 3.936699151992798, "mode": "train", "epoch": 14}
{"loss": 0.4998475909233093, "acc": 0.759368896484375, "length": 3.98046875, "mode": "test", "epoch": 14}
{"loss": 0.45113688707351685, "acc": 0.7858999967575073, "length": 3.936452865600586, "mode": "train", "epoch": 15}
{"loss": 0.49399423599243164, "acc": 0.7741333842277527, "length": 3.9921875, "mode": "test", "epoch": 15}
{"loss": 0.4486173689365387, "acc": 0.7917298674583435, "length": 3.956430196762085, "mode": "train", "epoch": 16}
{"loss": 0.48903900384902954, "acc": 0.77728271484375, "length": 3.98828125, "mode": "test", "epoch": 16}
{"loss": 0.4347890019416809, "acc": 0.8001950979232788, "length": 3.9555718898773193, "mode": "train", "epoch": 17}
{"loss": 0.48856300115585327, "acc": 0.7747558355331421, "length": 3.99609375, "mode": "test", "epoch": 17}
{"loss": 0.42136549949645996, "acc": 0.810918390750885, "length": 3.961872100830078, "mode": "train", "epoch": 18}
{"loss": 0.4684506058692932, "acc": 0.7801727056503296, "length": 3.99609375, "mode": "test", "epoch": 18}
{"loss": 0.4146672785282135, "acc": 0.8160174489021301, "length": 3.978402614593506, "mode": "train", "epoch": 19}
{"loss": 0.46221572160720825, "acc": 0.7839843034744263, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.40540003776550293, "acc": 0.8151801228523254, "length": 3.9781312942504883, "mode": "train", "epoch": 20}
{"loss": 0.44482117891311646, "acc": 0.7920868396759033, "length": 3.99609375, "mode": "test", "epoch": 20}
{"loss": 0.39815184473991394, "acc": 0.8236069083213806, "length": 3.975144386291504, "mode": "train", "epoch": 21}
{"loss": 0.4353134036064148, "acc": 0.7947264909744263, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.3850924074649811, "acc": 0.8298602104187012, "length": 3.9890713691711426, "mode": "train", "epoch": 22}
{"loss": 0.44161051511764526, "acc": 0.7982421517372131, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.3776508867740631, "acc": 0.8330923318862915, "length": 3.987689971923828, "mode": "train", "epoch": 23}
{"loss": 0.4353681802749634, "acc": 0.8031249046325684, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.3733977675437927, "acc": 0.8348897099494934, "length": 3.992050886154175, "mode": "train", "epoch": 24}
{"loss": 0.4522244930267334, "acc": 0.7994141578674316, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.36810559034347534, "acc": 0.8352839350700378, "length": 3.9927585124969482, "mode": "train", "epoch": 25}
{"loss": 0.4388335943222046, "acc": 0.8052734732627869, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.35942691564559937, "acc": 0.8448926210403442, "length": 3.996150493621826, "mode": "train", "epoch": 26}
{"loss": 0.43347811698913574, "acc": 0.8107422590255737, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.3671802580356598, "acc": 0.8375628590583801, "length": 3.9973654747009277, "mode": "train", "epoch": 27}
{"loss": 0.4407804012298584, "acc": 0.8033203482627869, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.355954110622406, "acc": 0.8404397964477539, "length": 3.9973056316375732, "mode": "train", "epoch": 28}
{"loss": 0.43648481369018555, "acc": 0.8000000715255737, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.3520871698856354, "acc": 0.8439307808876038, "length": 3.996619701385498, "mode": "train", "epoch": 29}
{"loss": 0.4266393780708313, "acc": 0.8056640625, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.3538481295108795, "acc": 0.8416172862052917, "length": 3.9990551471710205, "mode": "train", "epoch": 30}
{"loss": 0.4222085475921631, "acc": 0.8152343034744263, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.34577035903930664, "acc": 0.8476855754852295, "length": 3.9985854625701904, "mode": "train", "epoch": 31}
{"loss": 0.41559675335884094, "acc": 0.8166016340255737, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.3361237943172455, "acc": 0.8536946773529053, "length": 3.997755765914917, "mode": "train", "epoch": 32}
{"loss": 0.41037535667419434, "acc": 0.8189453482627869, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.3337370455265045, "acc": 0.8523643016815186, "length": 3.9996249675750732, "mode": "train", "epoch": 33}
{"loss": 0.39742958545684814, "acc": 0.8292969465255737, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.33603450655937195, "acc": 0.8516765832901001, "length": 3.9998292922973633, "mode": "train", "epoch": 34}
{"loss": 0.4067440629005432, "acc": 0.8187500238418579, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.3364351689815521, "acc": 0.8533467054367065, "length": 3.999516248703003, "mode": "train", "epoch": 35}
{"loss": 0.4104129374027252, "acc": 0.8201171159744263, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.32802608609199524, "acc": 0.855873703956604, "length": 3.999117851257324, "mode": "train", "epoch": 36}
{"loss": 0.4214010238647461, "acc": 0.8160156607627869, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.32525286078453064, "acc": 0.8597382307052612, "length": 3.9998397827148438, "mode": "train", "epoch": 37}
{"loss": 0.3918069005012512, "acc": 0.8240233659744263, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.3214479386806488, "acc": 0.859566867351532, "length": 3.9999122619628906, "mode": "train", "epoch": 38}
{"loss": 0.4059499502182007, "acc": 0.8195313811302185, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.31760498881340027, "acc": 0.8626234531402588, "length": 3.9991822242736816, "mode": "train", "epoch": 39}
{"loss": 0.4127585291862488, "acc": 0.8156250715255737, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.32418254017829895, "acc": 0.8598016500473022, "length": 3.9999749660491943, "mode": "train", "epoch": 40}
{"loss": 0.41373854875564575, "acc": 0.8162108063697815, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.32916566729545593, "acc": 0.855693519115448, "length": 3.9978511333465576, "mode": "train", "epoch": 41}
{"loss": 0.3956402540206909, "acc": 0.8248047232627869, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.3225017189979553, "acc": 0.8601420521736145, "length": 3.999819278717041, "mode": "train", "epoch": 42}
{"loss": 0.3982904553413391, "acc": 0.8347655534744263, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.3173200488090515, "acc": 0.8627609014511108, "length": 3.998872756958008, "mode": "train", "epoch": 43}
{"loss": 0.3861844539642334, "acc": 0.8375000357627869, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.31108900904655457, "acc": 0.8657151460647583, "length": 3.99989652633667, "mode": "train", "epoch": 44}
{"loss": 0.3886227607727051, "acc": 0.8333984017372131, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.31331056356430054, "acc": 0.8688225746154785, "length": 3.9989094734191895, "mode": "train", "epoch": 45}
{"loss": 0.405701607465744, "acc": 0.8275390267372131, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.305640310049057, "acc": 0.8708036541938782, "length": 3.9995920658111572, "mode": "train", "epoch": 46}
{"loss": 0.385292649269104, "acc": 0.8382812142372131, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.3110976219177246, "acc": 0.8696909546852112, "length": 3.9982423782348633, "mode": "train", "epoch": 47}
{"loss": 0.39338088035583496, "acc": 0.8390625715255737, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.31156685948371887, "acc": 0.8720986247062683, "length": 3.999953031539917, "mode": "train", "epoch": 48}
{"loss": 0.38390976190567017, "acc": 0.8359375, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.29687193036079407, "acc": 0.8791617751121521, "length": 3.9995837211608887, "mode": "train", "epoch": 49}
{"loss": 0.37264490127563477, "acc": 0.8541015982627869, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.28374332189559937, "acc": 0.8878481388092041, "length": 3.9995312690734863, "mode": "train", "epoch": 50}
{"loss": 0.3704652786254883, "acc": 0.8576172590255737, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.28514590859413147, "acc": 0.8872525095939636, "length": 3.998897075653076, "mode": "train", "epoch": 51}
{"loss": 0.3624184727668762, "acc": 0.8460937142372131, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.2843894064426422, "acc": 0.8845602869987488, "length": 3.999713897705078, "mode": "train", "epoch": 52}
{"loss": 0.35938340425491333, "acc": 0.8617187738418579, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.2742546796798706, "acc": 0.892613410949707, "length": 3.999636650085449, "mode": "train", "epoch": 53}
{"loss": 0.3529468774795532, "acc": 0.8658202886581421, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.26982349157333374, "acc": 0.8923341035842896, "length": 3.9998743534088135, "mode": "train", "epoch": 54}
{"loss": 0.3617933392524719, "acc": 0.8636718392372131, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.27010947465896606, "acc": 0.8942536115646362, "length": 3.999070405960083, "mode": "train", "epoch": 55}
{"loss": 0.3685201406478882, "acc": 0.8597655892372131, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.2676248252391815, "acc": 0.895979642868042, "length": 3.998542547225952, "mode": "train", "epoch": 56}
{"loss": 0.34991025924682617, "acc": 0.8650389909744263, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.2635118365287781, "acc": 0.9001733660697937, "length": 3.9999899864196777, "mode": "train", "epoch": 57}
{"loss": 0.3515070974826813, "acc": 0.8714843392372131, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.25321629643440247, "acc": 0.905451238155365, "length": 3.999983072280884, "mode": "train", "epoch": 58}
{"loss": 0.33917900919914246, "acc": 0.8738280534744263, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.25184765458106995, "acc": 0.9034773707389832, "length": 3.9999301433563232, "mode": "train", "epoch": 59}
{"loss": 0.33229860663414, "acc": 0.8761719465255737, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.2535420060157776, "acc": 0.9053360819816589, "length": 3.9999945163726807, "mode": "train", "epoch": 60}
{"loss": 0.3481210470199585, "acc": 0.8763671517372131, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.7313792705535889
